kaggle_environments/agent.py

```diff
@@ -99,8 +101,7 @@ def url_agent(observation, configuration):
 
 
 class Agent:
-    def __init__(self, raw, configuration, environment):
-        self.configuration = configuration
+    def __init__(self, raw, environment):
```

Comment:
configuration is passed as part of the environment anyway so it's pointless to pass as a separate variable